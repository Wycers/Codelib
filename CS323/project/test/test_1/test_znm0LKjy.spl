int test5(int a, int b)
{
    return a + b;
}
