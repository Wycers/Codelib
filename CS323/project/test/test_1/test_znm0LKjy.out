Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test5
        LP
        VarList (1)
          ParamDec (1)
            Specifier (1)
              TYPE: int
            VarDec (1)
              ID: a
          COMMA
          VarList (1)
            ParamDec (1)
              Specifier (1)
                TYPE: int
              VarDec (1)
                ID: b
        RP
      CompSt (2)
        LC
        StmtList (3)
          Stmt (3)
            RETURN
            Exp (3)
              Exp (3)
                ID: a
              PLUS
              Exp (3)
                ID: b
            SEMI
        RC
